cisc/cisc.go

```diff
@@ -357,7 +358,20 @@ func scQrcode(c *cli.Context) error {
 		return errors.New("Please chose one of the existing skipchain-ids")
 	}
 	scid := []byte(id.ID)
-	str := fmt.Sprintf("cisc://%s/%x", id.Data.Roster.RandomServerIdentity().Address.NetworkAddress(),
+	address := strings.Split(id.Data.Roster.RandomServerIdentity().Address.NetworkAddress(), ":")
+
+	if address[0] == "localhost" && c.Bool("e") {
+		conn, err := net.Dial("udp", "8.8.8.8:80")
+		if err != nil {
+			return err
+		}
+		defer conn.Close()
```

Comment:
you can even put the `conn.Close()` after the `localAddr :=`, as you don't need the connection anymore afterwards.